packages/controller-utils/src/util.test.ts

```diff
@@ -3,9 +3,9 @@ import BigNumber from 'bignumber.js';
 import BN from 'bn.js';
 import nock from 'nock';
 
-import { FakeProvider } from '../../../tests/fake-provider';
 import { MAX_SAFE_CHAIN_ID } from './constants';
 import * as util from './util';
+import { FakeProvider } from '../../../tests/fake-provider';
```

Comment:
The change of order made here would need to be reflected here by removing this line https://github.com/MetaMask/core/blob/91cabc2b1912a77074dbe07956df9b51163fb875/eslint-warning-thresholds.json#L168